discord/flags.py

```diff
@@ -754,12 +790,12 @@ class Intents(BaseFlags):
 
     __slots__ = ()
 
-    def __init__(self, value: int = 0, **kwargs: bool) -> None:
+    def __init__(self, value: int = 0, **kwargs: Unpack[_IntentsFlagsKwargs]) -> None:
```

Comment:
Another reason why I was not a fan of doing this is because now there are *more* places to edit when adding a single thing. This is especially egregious with permissions which need to have an added value both in its `TYPE_CHECKING` block and *now* in its `Unpack` dictionary.

I suppose it's unavoidable, but honestly it's just really annoying.